examples/components/components-access-child/index.html

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Angular 2 Components - NgFor</title>
+    <title>Angular 2 Component - Output</title>
     <script src="https://unpkg.com/systemjs@0.19.6/dist/system.src.js"></script>
     <script src="https://code.angularjs.org/tools/typescript.js"></script>
     <script src="system.config.js"></script>
   </head>
   <body>
-    <app>Loading...</app>
+    <rio-app>Loading...</rio-app>
 
     <script>
-      System.import('app');
+      System.import('main');
     </script>
   </body>
 
```

Comment:
For consistency, the main app file was renamed as `main.ts`
In the `system.config.js` file, that file was mapped as `main`, therefore it was changed the import call in index.html